common/djangoapps/enrollment/views.py

```diff
@@ -26,7 +29,11 @@
     OAuth2AuthenticationAllowInactiveUser,
     SessionAuthenticationAllowInactiveUser
 )
-from openedx.core.lib.api.permissions import ApiKeyHeaderPermission, ApiKeyHeaderPermissionIsAuthenticated
+from openedx.core.lib.api.permissions import (
+    ApiKeyHeaderPermission,
+    ApiKeyHeaderPermissionIsAuthenticated,
+    IsUserInUrlOrStaff
```

Comment:
What about using [IsStaffOrOwner](https://github.com/edx/edx-platform/blob/e8a36957b1f732974260e7b9b42b9c25148b492c/openedx/core/lib/api/permissions.py#L145) -- would be nice if we just had an IsStaff permission class, though.